frontend/src/components/managePermissions/table/ApplicationAdminTable.vue

```diff
@@ -29,6 +31,26 @@ const props = defineProps({
         type: [Array] as PropType<FamAppAdminGetResponse[] | undefined>,
         required: true,
     },
+    newAppAdminId: {
+        type: Number || undefined,
+        default: undefined,
+    },
+});
+
+// newAppAdminId goes to the top of the array
+const applicationAdmins = computed(() => {
```

Comment:
I feel we don't need to sort each table data in the table component, so all the data initially was got from `frontend/src/services/fetchData.ts`, we should sort them in those fetch methods.

Reply:
I will try to move it to the fetch file